package.json

```diff
@@ -6,8 +6,8 @@
   "scripts": {
     "example": "node example.js",
     "example-with-options": "node example-ejs-with-some-options.js",
-    "test-with-snapshot": "standard && cross-env TAP_SNAPSHOT=1 tap test-with-snapshot.js",
-    "test": "standard && tap test.js test-with-snapshot.js && tsc"
+    "test-with-snapshot": "standard && cross-env TAP_SNAPSHOT=1 tap test/test-ejs-with-snapshot.js",
+    "test": "standard && tap test/* && tsc"
```

Comment:
Pass a -J flag to tap. It will use all cpus to run multiple tests.

Reply:
👍 done.